File: iraf/unix/boot/spp/rpp/ratlibr/clower.r
include	defs

# clower - change letter to lower case

   character function clower(c)
   character c

   character k

   if (c >= BIGA & c <= BIGZ) {
      k = LETA - BIGA	# avoid integer overflow in byte machines
      clower = c + k
      }
   else
      clower = c

   return
   end

File: unix/boot/spp/rpp/ratlibf/clower.f
      integer function clower(c)
      integer c
      integer k
      if (.not.(c .ge. 65 .and. c .le. 90))goto 23000
      k = 97 - 65
      clower = c + k
      goto 23001
23000 continue
      clower = c
23001 continue
      return
      end
